files/__addonLocation__/babel.config.json

```diff
@@ -3,7 +3,7 @@
 <% } %>  "plugins": [<% if (typescript) { %>
     ["@babel/plugin-transform-typescript", { "allowDeclareFields": true }],<% } %>
     "@embroider/addon-dev/template-colocation-plugin",
-    ["@babel/plugin-proposal-decorators", { "legacy": true }],
+    ["@babel/plugin-proposal-decorators", { "decoratorsBeforeExport": true }],
```

Comment:
@void-mAlex @NullVoxPopuli When setting `decoratorsBeforeExport` to `true, does one need to configure their v2 addon somehow more?

My docs and test apps for `ember-container-query` failed: https://github.com/ijlee2/ember-container-query/actions/runs/5124925949

I couldn't find examples of other addons that currently use `decoratorsBeforeExport: true`.

https://emberobserver.com/code-search?codeQuery=decoratorsBeforeExport&fileFilter=babel

Reply:
Yes, I will create a PR, thanks for the quick feedback.

Reply:
https://github.com/embroider-build/addon-blueprint/pull/127